index.md

```diff
@@ -260,3 +309,32 @@ Licensed under [CC BY NC 4.0](https://creativecommons.org/licenses/by-nc/4.0/leg
 
 BrikWars is copyright ©1995-2010 Mike Rayhawk.
 For more information and the complete rules, please visit http://www.brikwars.com.
+
+---
+
+# (Intro?) What this game is and why we made it
+
+>Our Zombies Are Different
```

Comment:
I'd shorten this section to ~50% for ease of digestion and put the word parody in there somewhere - preferably instead of "avoid" in paragraph 3